functions/AutoRemediateS3-016.js

```diff
@@ -27,11 +28,36 @@ module.exports.handler = (event, context, callback) => {
   }
 
   let S3Bucket = event.resource
+  let retries = 5;
 
-  return UpdateBucketPolicy(S3Bucket).then(function () {
+  const retryable = function(S3Bucket) {
+
+    return UpdateBucketPolicy(S3Bucket).catch(function(err) {
+
+      console.log("Error found:");
+      console.error(err);
+      console.log("%d attempts remaining, retrying...", retries);
```

Comment:
Using the `logger` class here instead would be nice since we can toggle it 'on' and 'off' depending on the env